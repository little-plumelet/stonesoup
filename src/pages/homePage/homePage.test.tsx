import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import * as reactHooks from 'react-redux';
import { useAppDispatch, useAppSelector } from '../../hooks';
import { HomePage } from './HomePage';
import '../../../matchMedia';

jest.mock('react-redux');

const mockRecipeList = [
  {
    id: 716428,
    title: 'Pasta with Garlic, Scallions, Cauliflower & Breadcrumbs',
    image: 'https://spoonacular.com/recipeImages/716429-556x370.jpg',
    readyInMinutes: 15,
    aggregateLikes: 29,
  },
  {
    id: 716429,
    title: 'Risotto with mashrums',
    image: 'https://spoonacular.com/recipeImages/716429-556x370.jpg',
    readyInMinutes: 57,
    aggregateLikes: 84,
  },
];
const foo = { useAppSelector, useAppDispatch };
const dispatch = jest.fn();

jest.spyOn(foo, 'useAppDispatch').mockReturnValue(dispatch);
jest.spyOn(reactHooks, 'useDispatch').mockReturnValue(dispatch);

describe('Home page', () => {
  it('should render', () => {
    jest.spyOn(foo, 'useAppSelector').mockReturnValue({
      loading: false,
      error: null,
      recipes: [],
    });

    const component = render(
      <BrowserRouter>
        <HomePage />
      </BrowserRouter>
    );

    expect(component).toMatchSnapshot();
  });

  it('should render cards', async () => {
    jest.spyOn(foo, 'useAppSelector').mockReturnValue({
      loading: false,
      error: null,
      recipes: mockRecipeList,
    });

    render(
      <BrowserRouter>
        <HomePage />
      </BrowserRouter>
    );

    expect(screen.getAllByText('Risotto with mashrums')).toHaveLength(2);
  });
});
