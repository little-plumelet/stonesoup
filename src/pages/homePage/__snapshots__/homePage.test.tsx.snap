// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="ant-layout layout css-dev-only-do-not-override-mxhywb"
      >
        <header
          class="headerBlock"
        >
          <div
            class="header"
          >
            <div
              class="logoBlock"
            >
              <img
                alt="logo"
                class="logoImg"
                src=""
              />
              <h1
                class="logo"
              >
                StoneSoup
              </h1>
              <h1
                class="logoClone"
              >
                StoneSoup
              </h1>
            </div>
            <nav
              class="nav"
              id="nav"
            >
              <a
                aria-current="page"
                class="navLink, active"
                href="/"
              >
                Home
              </a>
              <a
                class="navLink, pending"
                href="/about"
              >
                About Us
              </a>
            </nav>
            <div
              class="ant-space css-dev-only-do-not-override-mxhywb ant-space-horizontal ant-space-align-center"
            >
              <div
                class="ant-space-item"
                style="margin-right: 8px;"
              >
                <button
                  class="ant-btn css-dev-only-do-not-override-mxhywb ant-btn-circle ant-btn-default ant-btn-icon-only"
                  data-testid="searchicon"
                  type="button"
                >
                  <span
                    aria-label="search"
                    class="anticon anticon-search"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="search"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                      />
                    </svg>
                  </span>
                </button>
              </div>
              <div
                class="ant-space-item"
              >
                <button
                  class="ant-btn css-dev-only-do-not-override-mxhywb ant-btn-circle ant-btn-default ant-btn-icon-only"
                  type="button"
                >
                  <span
                    aria-label="user"
                    class="anticon anticon-user"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="user"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </div>
          </div>
          <div
            class="searchBar"
          />
        </header>
        <main
          class="ant-layout-content content css-dev-only-do-not-override-mxhywb"
        >
          <div
            class="ant-card ant-card-bordered dayRecipeCard css-dev-only-do-not-override-mxhywb"
          >
            <div
              class="ant-card-body"
            >
              <div
                class="content"
              >
                <div
                  class="img"
                  style="background-image: url(https://spoonacular.com/recipeImages/632625-556x370.jpg); background-position: center; background-size: cover; background-repeat: no-repeat;"
                />
                <div
                  class="desc"
                >
                  <div>
                    <p>
                      Recipe of the day
                    </p>
                    <h2
                      class="title"
                    >
                      Apple-Cheesecake Tart with Salted Caramel Glaze
                    </h2>
                  </div>
                  <button
                    class="ant-btn css-dev-only-do-not-override-mxhywb ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only arrow"
                    data-testid="DayRecipeButton"
                    type="button"
                  >
                    <span
                      aria-label="arrow-right"
                      class="anticon anticon-arrow-right"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="arrow-right"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                        />
                      </svg>
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
          <div
            class="recipeBlocks"
          >
            <div>
              <h3
                class="ant-typography css-dev-only-do-not-override-mxhywb"
              >
                Sweet tooth
              </h3>
            </div>
            <div>
              <h3
                class="ant-typography css-dev-only-do-not-override-mxhywb"
              >
                Vegetarian
              </h3>
            </div>
          </div>
        </main>
        <footer
          class="footer"
        >
          <div
            class="footerBlock"
          >
            <div
              class="logoBlock"
            >
              <h2
                class="logo"
              >
                Stonesoup
              </h2>
              <p
                class="logoContent"
              >
                Indulge in culinary magic with tantalizing recipes and the enchanting tale of Stone Soup, where flavors blend like a fairytale come true.
              </p>
            </div>
            <div
              class="navBlock"
            >
              <nav
                class="nav"
              >
                <p>
                  Stonesoup
                </p>
                <a
                  class="link"
                  href="/"
                >
                  Home
                </a>
                <a
                  class="link"
                  href="/about"
                >
                  About Us
                </a>
                <a
                  class="link"
                  href="/"
                >
                  Contacts
                </a>
              </nav>
              <nav
                class="nav"
              >
                <p>
                  Follow
                </p>
                <a
                  class="link"
                  href="https://www.instagram.com"
                >
                  Instagram
                </a>
                <a
                  class="link"
                  href="https://www.facebook.com"
                >
                  Facebook
                </a>
                <a
                  class="link"
                  href="https://www.youtube.com"
                >
                  Youtube
                </a>
              </nav>
            </div>
          </div>
          <div
            class="ant-divider css-dev-only-do-not-override-mxhywb ant-divider-horizontal"
            role="separator"
          />
          <div
            class="footerBlock"
          >
            <p
              class="copyright"
            >
              Â© 2023 Stonesoup - all rights reserved
            </p>
            <div
              class="socialNetBlock"
            >
              <span
                aria-label="instagram"
                class="anticon anticon-instagram icon"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="instagram"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 306.9c-113.5 0-205.1 91.6-205.1 205.1S398.5 717.1 512 717.1 717.1 625.5 717.1 512 625.5 306.9 512 306.9zm0 338.4c-73.4 0-133.3-59.9-133.3-133.3S438.6 378.7 512 378.7 645.3 438.6 645.3 512 585.4 645.3 512 645.3zm213.5-394.6c-26.5 0-47.9 21.4-47.9 47.9s21.4 47.9 47.9 47.9 47.9-21.3 47.9-47.9a47.84 47.84 0 00-47.9-47.9zM911.8 512c0-55.2.5-109.9-2.6-165-3.1-64-17.7-120.8-64.5-167.6-46.9-46.9-103.6-61.4-167.6-64.5-55.2-3.1-109.9-2.6-165-2.6-55.2 0-109.9-.5-165 2.6-64 3.1-120.8 17.7-167.6 64.5C132.6 226.3 118.1 283 115 347c-3.1 55.2-2.6 109.9-2.6 165s-.5 109.9 2.6 165c3.1 64 17.7 120.8 64.5 167.6 46.9 46.9 103.6 61.4 167.6 64.5 55.2 3.1 109.9 2.6 165 2.6 55.2 0 109.9.5 165-2.6 64-3.1 120.8-17.7 167.6-64.5 46.9-46.9 61.4-103.6 64.5-167.6 3.2-55.1 2.6-109.8 2.6-165zm-88 235.8c-7.3 18.2-16.1 31.8-30.2 45.8-14.1 14.1-27.6 22.9-45.8 30.2C695.2 844.7 570.3 840 512 840c-58.3 0-183.3 4.7-235.9-16.1-18.2-7.3-31.8-16.1-45.8-30.2-14.1-14.1-22.9-27.6-30.2-45.8C179.3 695.2 184 570.3 184 512c0-58.3-4.7-183.3 16.1-235.9 7.3-18.2 16.1-31.8 30.2-45.8s27.6-22.9 45.8-30.2C328.7 179.3 453.7 184 512 184s183.3-4.7 235.9 16.1c18.2 7.3 31.8 16.1 45.8 30.2 14.1 14.1 22.9 27.6 30.2 45.8C844.7 328.7 840 453.7 840 512c0 58.3 4.7 183.2-16.2 235.8z"
                  />
                </svg>
              </span>
              <span
                aria-label="facebook"
                class="anticon anticon-facebook icon"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="facebook"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-32 736H663.9V602.2h104l15.6-120.7H663.9v-77.1c0-35 9.7-58.8 59.8-58.8h63.9v-108c-11.1-1.5-49-4.8-93.2-4.8-92.2 0-155.3 56.3-155.3 159.6v89H434.9v120.7h104.3V848H176V176h672v672z"
                  />
                </svg>
              </span>
              <span
                aria-label="youtube"
                class="anticon anticon-youtube icon"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="youtube"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M960 509.2c0-2.2 0-4.7-.1-7.6-.1-8.1-.3-17.2-.5-26.9-.8-27.9-2.2-55.7-4.4-81.9-3-36.1-7.4-66.2-13.4-88.8a139.52 139.52 0 00-98.3-98.5c-28.3-7.6-83.7-12.3-161.7-15.2-37.1-1.4-76.8-2.3-116.5-2.8-13.9-.2-26.8-.3-38.4-.4h-29.4c-11.6.1-24.5.2-38.4.4-39.7.5-79.4 1.4-116.5 2.8-78 3-133.5 7.7-161.7 15.2A139.35 139.35 0 0082.4 304C76.3 326.6 72 356.7 69 392.8c-2.2 26.2-3.6 54-4.4 81.9-.3 9.7-.4 18.8-.5 26.9 0 2.9-.1 5.4-.1 7.6v5.6c0 2.2 0 4.7.1 7.6.1 8.1.3 17.2.5 26.9.8 27.9 2.2 55.7 4.4 81.9 3 36.1 7.4 66.2 13.4 88.8 12.8 47.9 50.4 85.7 98.3 98.5 28.2 7.6 83.7 12.3 161.7 15.2 37.1 1.4 76.8 2.3 116.5 2.8 13.9.2 26.8.3 38.4.4h29.4c11.6-.1 24.5-.2 38.4-.4 39.7-.5 79.4-1.4 116.5-2.8 78-3 133.5-7.7 161.7-15.2 47.9-12.8 85.5-50.5 98.3-98.5 6.1-22.6 10.4-52.7 13.4-88.8 2.2-26.2 3.6-54 4.4-81.9.3-9.7.4-18.8.5-26.9 0-2.9.1-5.4.1-7.6v-5.6zm-72 5.2c0 2.1 0 4.4-.1 7.1-.1 7.8-.3 16.4-.5 25.7-.7 26.6-2.1 53.2-4.2 77.9-2.7 32.2-6.5 58.6-11.2 76.3-6.2 23.1-24.4 41.4-47.4 47.5-21 5.6-73.9 10.1-145.8 12.8-36.4 1.4-75.6 2.3-114.7 2.8-13.7.2-26.4.3-37.8.3h-28.6l-37.8-.3c-39.1-.5-78.2-1.4-114.7-2.8-71.9-2.8-124.9-7.2-145.8-12.8-23-6.2-41.2-24.4-47.4-47.5-4.7-17.7-8.5-44.1-11.2-76.3-2.1-24.7-3.4-51.3-4.2-77.9-.3-9.3-.4-18-.5-25.7 0-2.7-.1-5.1-.1-7.1v-4.8c0-2.1 0-4.4.1-7.1.1-7.8.3-16.4.5-25.7.7-26.6 2.1-53.2 4.2-77.9 2.7-32.2 6.5-58.6 11.2-76.3 6.2-23.1 24.4-41.4 47.4-47.5 21-5.6 73.9-10.1 145.8-12.8 36.4-1.4 75.6-2.3 114.7-2.8 13.7-.2 26.4-.3 37.8-.3h28.6l37.8.3c39.1.5 78.2 1.4 114.7 2.8 71.9 2.8 124.9 7.2 145.8 12.8 23 6.2 41.2 24.4 47.4 47.5 4.7 17.7 8.5 44.1 11.2 76.3 2.1 24.7 3.4 51.3 4.2 77.9.3 9.3.4 18 .5 25.7 0 2.7.1 5.1.1 7.1v4.8zM423 646l232-135-232-133z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </footer>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="ant-layout layout css-dev-only-do-not-override-mxhywb"
    >
      <header
        class="headerBlock"
      >
        <div
          class="header"
        >
          <div
            class="logoBlock"
          >
            <img
              alt="logo"
              class="logoImg"
              src=""
            />
            <h1
              class="logo"
            >
              StoneSoup
            </h1>
            <h1
              class="logoClone"
            >
              StoneSoup
            </h1>
          </div>
          <nav
            class="nav"
            id="nav"
          >
            <a
              aria-current="page"
              class="navLink, active"
              href="/"
            >
              Home
            </a>
            <a
              class="navLink, pending"
              href="/about"
            >
              About Us
            </a>
          </nav>
          <div
            class="ant-space css-dev-only-do-not-override-mxhywb ant-space-horizontal ant-space-align-center"
          >
            <div
              class="ant-space-item"
              style="margin-right: 8px;"
            >
              <button
                class="ant-btn css-dev-only-do-not-override-mxhywb ant-btn-circle ant-btn-default ant-btn-icon-only"
                data-testid="searchicon"
                type="button"
              >
                <span
                  aria-label="search"
                  class="anticon anticon-search"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="search"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                    />
                  </svg>
                </span>
              </button>
            </div>
            <div
              class="ant-space-item"
            >
              <button
                class="ant-btn css-dev-only-do-not-override-mxhywb ant-btn-circle ant-btn-default ant-btn-icon-only"
                type="button"
              >
                <span
                  aria-label="user"
                  class="anticon anticon-user"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="user"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          class="searchBar"
        />
      </header>
      <main
        class="ant-layout-content content css-dev-only-do-not-override-mxhywb"
      >
        <div
          class="ant-card ant-card-bordered dayRecipeCard css-dev-only-do-not-override-mxhywb"
        >
          <div
            class="ant-card-body"
          >
            <div
              class="content"
            >
              <div
                class="img"
                style="background-image: url(https://spoonacular.com/recipeImages/632625-556x370.jpg); background-position: center; background-size: cover; background-repeat: no-repeat;"
              />
              <div
                class="desc"
              >
                <div>
                  <p>
                    Recipe of the day
                  </p>
                  <h2
                    class="title"
                  >
                    Apple-Cheesecake Tart with Salted Caramel Glaze
                  </h2>
                </div>
                <button
                  class="ant-btn css-dev-only-do-not-override-mxhywb ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only arrow"
                  data-testid="DayRecipeButton"
                  type="button"
                >
                  <span
                    aria-label="arrow-right"
                    class="anticon anticon-arrow-right"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="arrow-right"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M869 487.8L491.2 159.9c-2.9-2.5-6.6-3.9-10.5-3.9h-88.5c-7.4 0-10.8 9.2-5.2 14l350.2 304H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h585.1L386.9 854c-5.6 4.9-2.2 14 5.2 14h91.5c1.9 0 3.8-.7 5.2-2L869 536.2a32.07 32.07 0 000-48.4z"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="recipeBlocks"
        >
          <div>
            <h3
              class="ant-typography css-dev-only-do-not-override-mxhywb"
            >
              Sweet tooth
            </h3>
          </div>
          <div>
            <h3
              class="ant-typography css-dev-only-do-not-override-mxhywb"
            >
              Vegetarian
            </h3>
          </div>
        </div>
      </main>
      <footer
        class="footer"
      >
        <div
          class="footerBlock"
        >
          <div
            class="logoBlock"
          >
            <h2
              class="logo"
            >
              Stonesoup
            </h2>
            <p
              class="logoContent"
            >
              Indulge in culinary magic with tantalizing recipes and the enchanting tale of Stone Soup, where flavors blend like a fairytale come true.
            </p>
          </div>
          <div
            class="navBlock"
          >
            <nav
              class="nav"
            >
              <p>
                Stonesoup
              </p>
              <a
                class="link"
                href="/"
              >
                Home
              </a>
              <a
                class="link"
                href="/about"
              >
                About Us
              </a>
              <a
                class="link"
                href="/"
              >
                Contacts
              </a>
            </nav>
            <nav
              class="nav"
            >
              <p>
                Follow
              </p>
              <a
                class="link"
                href="https://www.instagram.com"
              >
                Instagram
              </a>
              <a
                class="link"
                href="https://www.facebook.com"
              >
                Facebook
              </a>
              <a
                class="link"
                href="https://www.youtube.com"
              >
                Youtube
              </a>
            </nav>
          </div>
        </div>
        <div
          class="ant-divider css-dev-only-do-not-override-mxhywb ant-divider-horizontal"
          role="separator"
        />
        <div
          class="footerBlock"
        >
          <p
            class="copyright"
          >
            Â© 2023 Stonesoup - all rights reserved
          </p>
          <div
            class="socialNetBlock"
          >
            <span
              aria-label="instagram"
              class="anticon anticon-instagram icon"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="instagram"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M512 306.9c-113.5 0-205.1 91.6-205.1 205.1S398.5 717.1 512 717.1 717.1 625.5 717.1 512 625.5 306.9 512 306.9zm0 338.4c-73.4 0-133.3-59.9-133.3-133.3S438.6 378.7 512 378.7 645.3 438.6 645.3 512 585.4 645.3 512 645.3zm213.5-394.6c-26.5 0-47.9 21.4-47.9 47.9s21.4 47.9 47.9 47.9 47.9-21.3 47.9-47.9a47.84 47.84 0 00-47.9-47.9zM911.8 512c0-55.2.5-109.9-2.6-165-3.1-64-17.7-120.8-64.5-167.6-46.9-46.9-103.6-61.4-167.6-64.5-55.2-3.1-109.9-2.6-165-2.6-55.2 0-109.9-.5-165 2.6-64 3.1-120.8 17.7-167.6 64.5C132.6 226.3 118.1 283 115 347c-3.1 55.2-2.6 109.9-2.6 165s-.5 109.9 2.6 165c3.1 64 17.7 120.8 64.5 167.6 46.9 46.9 103.6 61.4 167.6 64.5 55.2 3.1 109.9 2.6 165 2.6 55.2 0 109.9.5 165-2.6 64-3.1 120.8-17.7 167.6-64.5 46.9-46.9 61.4-103.6 64.5-167.6 3.2-55.1 2.6-109.8 2.6-165zm-88 235.8c-7.3 18.2-16.1 31.8-30.2 45.8-14.1 14.1-27.6 22.9-45.8 30.2C695.2 844.7 570.3 840 512 840c-58.3 0-183.3 4.7-235.9-16.1-18.2-7.3-31.8-16.1-45.8-30.2-14.1-14.1-22.9-27.6-30.2-45.8C179.3 695.2 184 570.3 184 512c0-58.3-4.7-183.3 16.1-235.9 7.3-18.2 16.1-31.8 30.2-45.8s27.6-22.9 45.8-30.2C328.7 179.3 453.7 184 512 184s183.3-4.7 235.9 16.1c18.2 7.3 31.8 16.1 45.8 30.2 14.1 14.1 22.9 27.6 30.2 45.8C844.7 328.7 840 453.7 840 512c0 58.3 4.7 183.2-16.2 235.8z"
                />
              </svg>
            </span>
            <span
              aria-label="facebook"
              class="anticon anticon-facebook icon"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="facebook"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-32 736H663.9V602.2h104l15.6-120.7H663.9v-77.1c0-35 9.7-58.8 59.8-58.8h63.9v-108c-11.1-1.5-49-4.8-93.2-4.8-92.2 0-155.3 56.3-155.3 159.6v89H434.9v120.7h104.3V848H176V176h672v672z"
                />
              </svg>
            </span>
            <span
              aria-label="youtube"
              class="anticon anticon-youtube icon"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="youtube"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M960 509.2c0-2.2 0-4.7-.1-7.6-.1-8.1-.3-17.2-.5-26.9-.8-27.9-2.2-55.7-4.4-81.9-3-36.1-7.4-66.2-13.4-88.8a139.52 139.52 0 00-98.3-98.5c-28.3-7.6-83.7-12.3-161.7-15.2-37.1-1.4-76.8-2.3-116.5-2.8-13.9-.2-26.8-.3-38.4-.4h-29.4c-11.6.1-24.5.2-38.4.4-39.7.5-79.4 1.4-116.5 2.8-78 3-133.5 7.7-161.7 15.2A139.35 139.35 0 0082.4 304C76.3 326.6 72 356.7 69 392.8c-2.2 26.2-3.6 54-4.4 81.9-.3 9.7-.4 18.8-.5 26.9 0 2.9-.1 5.4-.1 7.6v5.6c0 2.2 0 4.7.1 7.6.1 8.1.3 17.2.5 26.9.8 27.9 2.2 55.7 4.4 81.9 3 36.1 7.4 66.2 13.4 88.8 12.8 47.9 50.4 85.7 98.3 98.5 28.2 7.6 83.7 12.3 161.7 15.2 37.1 1.4 76.8 2.3 116.5 2.8 13.9.2 26.8.3 38.4.4h29.4c11.6-.1 24.5-.2 38.4-.4 39.7-.5 79.4-1.4 116.5-2.8 78-3 133.5-7.7 161.7-15.2 47.9-12.8 85.5-50.5 98.3-98.5 6.1-22.6 10.4-52.7 13.4-88.8 2.2-26.2 3.6-54 4.4-81.9.3-9.7.4-18.8.5-26.9 0-2.9.1-5.4.1-7.6v-5.6zm-72 5.2c0 2.1 0 4.4-.1 7.1-.1 7.8-.3 16.4-.5 25.7-.7 26.6-2.1 53.2-4.2 77.9-2.7 32.2-6.5 58.6-11.2 76.3-6.2 23.1-24.4 41.4-47.4 47.5-21 5.6-73.9 10.1-145.8 12.8-36.4 1.4-75.6 2.3-114.7 2.8-13.7.2-26.4.3-37.8.3h-28.6l-37.8-.3c-39.1-.5-78.2-1.4-114.7-2.8-71.9-2.8-124.9-7.2-145.8-12.8-23-6.2-41.2-24.4-47.4-47.5-4.7-17.7-8.5-44.1-11.2-76.3-2.1-24.7-3.4-51.3-4.2-77.9-.3-9.3-.4-18-.5-25.7 0-2.7-.1-5.1-.1-7.1v-4.8c0-2.1 0-4.4.1-7.1.1-7.8.3-16.4.5-25.7.7-26.6 2.1-53.2 4.2-77.9 2.7-32.2 6.5-58.6 11.2-76.3 6.2-23.1 24.4-41.4 47.4-47.5 21-5.6 73.9-10.1 145.8-12.8 36.4-1.4 75.6-2.3 114.7-2.8 13.7-.2 26.4-.3 37.8-.3h28.6l37.8.3c39.1.5 78.2 1.4 114.7 2.8 71.9 2.8 124.9 7.2 145.8 12.8 23 6.2 41.2 24.4 47.4 47.5 4.7 17.7 8.5 44.1 11.2 76.3 2.1 24.7 3.4 51.3 4.2 77.9.3 9.3.4 18 .5 25.7 0 2.7.1 5.1.1 7.1v4.8zM423 646l232-135-232-133z"
                />
              </svg>
            </span>
          </div>
        </div>
      </footer>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
